composer.json

```diff
@@ -28,7 +28,7 @@
 		"phpseclib/phpseclib": "*",
 		"rackspace/php-opencloud": "v1.9.2",
 		"james-heinrich/getid3": "dev-master",
-		"ircmaxell/random-lib": "v1.0.0",
+		"ircmaxell/random-lib": "~1.1",
```

Comment:
hmmm - shouldn't we point to explicit versions? I'm still uncertain on how to do this "properly"

@bantu @nickvergessen